_assets/scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+source "${GIT_ROOT}/_assets/scripts/colors.sh"
+source "${GIT_ROOT}/_assets/scripts/codecov.sh"
+
+echo -e "${GRN}Running integration tests${RST}"
+
+root_path="${GIT_ROOT}/integration-tests"
+coverage_reports_path="${root_path}/coverage"
+test_results_path="${root_path}/reports"
+
+# Create directories
+mkdir -p "${GIT_ROOT}/integration-tests/coverage"
+
+# Cleanup any previous coverage reports
+rm -rf "${coverage_reports_path}"
```

Comment:
You are removing the folder right after you create it. Would be clearer if you used `coverage_reports_path` in the `mkdir`.

Reply:
Since the folder doesn't exist when then the volume for `status-go` containers is create it is owned by default by `root`.

Reply:
I recommend adding `set -x` to scripts to debug what the do:
```
Running integration tests
+ root_path=/home/jakubgs/go/src/github.com/status-im/status-go/integration-tests
+ coverage_reports_path=/home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/coverage
+ test_results_path=/home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/reports
+ mkdir -p /home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/coverage
+ ls -ld /home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/coverage
drwxr-xr-x 2 jakubgs jakubgs 2 Sep 12 19:04 /home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/coverage
+ rm -rf /home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/coverage
+ rm -rf /home/jakubgs/go/src/github.com/status-im/status-go/integration-tests/reports
```
That way you can clearly see you delete what you just created.